client/core/simharness/simharness.go

```diff
@@ -0,0 +1,474 @@
+	Dir      = filepath.Join(usr.HomeDir, "dextest")
+	CertPath = filepath.Join(Dir, "dcrdex", "rpc.cert")
+	Pass     = []byte("abc")
+	Host     = "127.0.0.1:17273"
+
+	dcrID, _   = dex.BipSymbolID("dcr")
+	btcID, _   = dex.BipSymbolID("btc")
+	ltcID, _   = dex.BipSymbolID("ltc")
+	ethID, _   = dex.BipSymbolID("eth")
+	bchID, _   = dex.BipSymbolID("bch")
+	zecID, _   = dex.BipSymbolID("zec")
+	dogeID, _  = dex.BipSymbolID("doge")
+	dexttID, _ = dex.BipSymbolID("dextt.eth")
+)
+
+const walletNameLength = 6
+
+var chars = []byte("123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz")
+
+func RandomWalletName() string {
+	b := make([]byte, walletNameLength)
+	for i := range b {
+		b[i] = chars[rand.Intn(len(chars))]
+	}
+	return string(b)
+}
+
+func RPCWalletForm(symbol, node, name, port string) (form, parentForm *core.WalletForm) {
+	switch symbol {
+	case "dcr":
+		form = &core.WalletForm{
+			Type:    "dcrwalletRPC",
+			AssetID: dcrID,
+			Config: map[string]string{
+				"account":   name,
+				"username":  "user",
+				"password":  "pass",
+				"rpccert":   filepath.Join(Dir, "dcr/"+node+"/rpc.cert"),
+				"rpclisten": port,
+			},
+		}
+	case "btc":
+		form = &core.WalletForm{
+			Type:    "bitcoindRPC",
+			AssetID: btcID,
+			Config: map[string]string{
+				"walletname":  name,
+				"rpcuser":     "user",
+				"rpcpassword": "pass",
+				"rpcport":     port,
+			},
+		}
+	case "ltc":
+		form = &core.WalletForm{
+			Type:    "litecoindRPC",
+			AssetID: ltcID,
+			Config: map[string]string{
+				"walletname":  name,
+				"rpcuser":     "user",
+				"rpcpassword": "pass",
+				"rpcport":     port,
+			},
+		}
+	case "bch":
+		form = &core.WalletForm{
+			Type:    "bitcoindRPC",
+			AssetID: bchID,
+			Config: map[string]string{
+				"walletname":  name,
+				"rpcuser":     "user",
+				"rpcpassword": "pass",
+				"rpcport":     port,
+			},
+		}
+	case "zec":
+		form = &core.WalletForm{
+			Type:    "zcashdRPC",
+			AssetID: zecID,
+			Config: map[string]string{
+				"walletname":  name,
+				"rpcuser":     "user",
+				"rpcpassword": "pass",
+				"rpcport":     port,
+			},
+		}
+	case "doge":
+		form = &core.WalletForm{
+			Type:    "dogecoindRPC",
+			AssetID: dogeID,
+			Config: map[string]string{
+				"walletname":  name,
+				"rpcuser":     "user",
+				"rpcpassword": "pass",
+				"rpcport":     port,
+			},
+		}
+	case "eth", "dextt.eth":
+		form = &core.WalletForm{
+			Type:    "geth",
+			AssetID: ethID,
+			Config: map[string]string{
+				"walletname":  name,
+				"rpcuser":     "user",
+				"rpcpassword": "pass",
+				"rpcport":     port,
+			},
+		}
+		if symbol == "dextt.eth" {
+			parentForm = form
+			form = &core.WalletForm{
+				Type:       "token",
+				AssetID:    dexttID,
+				ParentForm: form,
+			}
+		}
+	}
+	return
+}
+
+var zecSendMtx sync.RWMutex
+
+// Mine will mine a single block on the node and asset indicated.
+func Mine(ctx context.Context, symbol, node string) <-chan *HarnessResult {
```

Comment:
Unused.  Thinking to use this in other spots like loadbot and eth/nodeclient_harness_test.go?

About 90 loc in `RPCWalletForm` also unused.  Should all of this simharness stuff be in this PR?